src/test/scala/mesosphere/marathon/api/akkahttp/v2/ResponseMatchers.scala

```diff
@@ -87,6 +88,18 @@ trait ResponseMatchers { this: UnitTest =>
     }
   }
 
+  /**
+    * Have property matcher for id (e.g. of an app)
+    * @param pathId id of an app
+    * @return Match result
+    */
+  def id(pathId: PathId) = new HavePropertyMatcher[JsValue, Option[PathId]] {
```

Comment:
I've just noticed that this is better than `podId`. Would you mind replacing `have(podId)` with `have(id)`?